docs-guides/source/model-prediction.md

```diff
@@ -160,3 +160,140 @@ img_as_np_array = np.reshape(img_as_np_array, model_expected_input_shape)
 out_dict = model.predict({'image': img_as_np_array})
 ```
 
+
+## Using Compiled Python Models for Prediction
+
+You can use a *compiled* Core ML model to initialize the [MLModel](mlmodel) object for making predictions. For large models, using a compiled model can save considerable time in initializing the `mlmodel` object. 
```

Comment:
This statement is incorrect. You can't initialize an MLModel object with a compiled Core ML Model.

Reply:
OK, edited to this:

You can use a *compiled* Core ML model ([CompiledMLModel](https://apple.github.io/coremltools/source/coremltools.models.html#coremltools.models.CompiledMLModel)) rather than an [MLModel](https://apple.github.io/coremltools/source/coremltools.models.html#coremltools.models.model.MLModel) for making predictions.